lib/main.dart

```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
+                        children: <Widget>[
+                          new TextFormField(
+                            decoration: new InputDecoration(
+                              labelText: "Email",
+                            ),
+                            keyboardType: TextInputType.emailAddress,
+                          ),
+                          new TextFormField(
+                            decoration: new InputDecoration(
+                              labelText: "Password",
+                            ),
+                            keyboardType: TextInputType.text,
+                            obscureText: true,
+                          ),
+                          new Padding(
+                              padding: const EdgeInsets.only(
+                                top: 20.0
+                              ),
+                          ),
+                          new MaterialButton(
+                            color: Colors.teal,
+                            textColor: Colors.white,
+                            child: new Text("Login"),
+                            onPressed: () => {},
+                          )
+                        ],
+                      ),
+                    ),
+                  ),
+              )
+            ],
+          )
+        ],
       ),
-      body: new Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: new Column(
-          // Column is also layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug paint" (press "p" in the console where you ran
-          // "flutter run", or select "Toggle Debug Paint" from the Flutter tool
-          // window in IntelliJ) to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            new Text(
-              'You have pushed the button this many times:',
-            ),
-            new Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.display1,
-            ),
-          ],
-        ),
-      ),
-      floatingActionButton: new FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: new Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
     );
   }
-}
+}
```

Comment:
Convert it to windows file format as it was. Look for how to do it for specific editor/platform you are using and update it here also for others.
Refer this: https://kb.iu.edu/d/acux

Reply:
What is this for? I am using android studio and I can export my formatter.

Reply:
The repo must have been originally coded on unix system where new line does not contain <enter> ASCII char. And now when you saved it through windows system, it added it. Thus though it is not visible, there is change in this line.